app/controllers/DefaultsController.scala

```diff
@@ -30,11 +30,13 @@ case class Defaults(
   collectionCap: Int,
   navListCap: Int,
   navListType: String,
-  collectionMetadata: Iterable[Metadata]
+  collectionMetadata: Iterable[Metadata],
+  capiLiveUrl: String,
+  capiPreviewUrl: String
```

Comment:
If you make these urls optional, then you don't need to include them in the v1 config, you can only send them back to the v2 app.

Reply:
Good plan 👍